chart/templates/deployment-sidekiq.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         {{- end }}
         # roll the pods to pick up any db migrations or other changes
         {{- include "mastodon.rollingPodAnnotations" . | nindent 8 }}
+        checksum/config-secrets-smtp: {{ include ( print $.Template.BasePath "/secret-smtp.yaml" ) . | sha256sum | quote }}
```

Comment:
Does this work properly if `existingSecret` is set?

Reply:
Yes, I tested it to be sure. The file returns an empty string which is hashed normally.

It might not be the most elegant thing to do though. Would you prefer or me to guard the annotation behind an `if`?